### ISSUE TITLE
Integrate convolutional neural network quantum circuits with exisiting interfaces.

### ISSUE THREAD

**Issue by dlasecki** [Qiskit/qiskit-machine-learning#293]
The goal of this issue is to make use of existing interfaces for neural networks and reuse loss functions, optimizers etc. for convolutional quantum neural networks.

**Comment by adekusar-drl** [comment]
Potentially can be implemented in #408.

**Comment by adekusar-drl** [comment]
Closed via #462
